influxdb_store.go

```diff
@@ -391,7 +448,12 @@ func (in *InfluxDBStore) Traces(opts TracesOpts) ([]*Trace, error) {
 	return traces, nil
 }
 
+// Close flushes the last batch to InfluxDB and shuts down the InfluxDBStore.
 func (in *InfluxDBStore) Close() error {
+	close(in.flusherStopChan)
+	if err := in.flush(); err != nil {
+		return err
```

Comment:
should `Close` return an error if flush fails? I'd expect it to have a similiar behaviour as above, and just cause a log. Also as this stands, if the flush fails, the server doesn't close.